gradle.properties

```diff
@@ -7,10 +7,10 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
 
 springBootVersion=2.1.5.RELEASE
 micronautVersion=1.2.4
-eventuateTramVersion=0.23.0.MICRONAUT.M5
+eventuateTramVersion=0.22.0-SNAPSHOT
```

Comment:
Why SNAPSHOT versions?
New versions of the dependencies have been released

Reply:
I will fix